frontend-react/src/__mocks__/HistoryMockServer.ts

```diff
@@ -4,6 +4,25 @@ import { setupServer } from "msw/node";
 import ReportResource from "../resources/ReportResource";
 
 const handlers = [
+    rest.get(
+        "https://test.prime.cdc.gov/api/history/report",
+        (req, res, ctx) => {
+            if (
+                !req.headers.get("authorization")?.includes("TOKEN") ||
+                !req.headers.get("organization")
```

Comment:
I'll double check, but some places I believe I do specifically want to test the value of the header to return a conditional response, and this might just be a copy-paste of that. If this _is_ the spot where I care about the value, I would keep it, but yeah, otherwise, I only care that it came through.